botbot/Module/OpenAIModule.cs

```diff
@@ -24,26 +25,49 @@ public OpenAIModule()
             encoder = Encoder.Get_Encoder();
         }
 
+        private string GetPrefix(string text) {
+            string[] splitText = text.Split(' ');
+            string command = splitText[1];
+            if (String.Equals(command, "eli5") ){
+                return "Explain like I'm 5 years old: ";
+            }
+
+            return "";
+        }
+
+        private string GetPrompt(string text) {
+            string[] splitText = text.Split(' ');
+            if (splitText[2].StartsWith("temp")) {
+                return text.Substring(
```

Comment:
If you like how these methods are split up then I can clean this up to be less ugly

Reply:
Yeah it should be formatted in the "C#" way

Reply:
I'd need to use your config because my default prettier config formats different from the patterns in this repo

Reply:
I can fix it in post.